lookout/style/typos/analyzer.py

```diff
@@ -21,12 +20,14 @@
 from lookout.style.typos.corrector_manager import TyposCorrectorManager
 from lookout.style.typos.utils import Candidate, Columns, flatten_df_by_column, TEMPLATE_DIR
 
-TypoFix = NamedTuple("LineFix", (
-    ("head_file", File),  # file from head revision
-    ("line_number", int),  # line number for the comment
-    ("identifier", str),  # identifier where typo is found
-    ("candidates", Mapping[str, Iterable[Candidate]]),  # Candidates for token {token: candidates}
-    ("identifier_candidates", Iterable[Candidate]),  # Suggested identifiers
+TypoFix = NamedTuple("TypoFix", (
+    ("content", str),                                         # file content from head revision
+    ("path", str),                                            # file path from head revision
+    ("line_number", int),                                     # line number for the comment
+    ("identifier", str),                                      # identifier where typo is found
+    ("token_candidates", Mapping[str, Iterable[Candidate]]),  # candidates for token
+                                                              # {token: candidates}
+    ("identifier_candidates", Iterable[Candidate]),           # suggested identifiers
```

Comment:
Can you build `identifier_candidates` from `token_candidates`? If you can, then better to inherit and implement the getter.

Reply:
well, I actually do not need `"token_candidates", let's just drop it and keep `identifier_candidates` only?

Reply:
That's what the original code was before somebody added the mapping. OK, let's remove then

Reply:
And name it "canidates"